library/spdm_responder_lib/libspdm_rsp_psk_exchange.c

```diff
@@ -198,6 +198,15 @@ libspdm_return_t libspdm_get_response_psk_exchange(void *context,
     spdm_response->header.request_response_code = SPDM_PSK_EXCHANGE_RSP;
     spdm_response->header.param1 = spdm_context->local_context.heartbeat_period;
 
+    if (spdm_request->psk_hint_length == 0) {
+        spdm_context->local_context.psk_hint_size = 0;
+        spdm_context->local_context.psk_hint = NULL;
+    } else {
+        spdm_context->local_context.psk_hint_size = spdm_request->psk_hint_length;
+        spdm_context->local_context.psk_hint = (const uint8_t *)request +
+                                               sizeof(spdm_psk_exchange_request_t);
+    }
```

Comment:
local_context is designed to be provisioned by integrator.

Here we need a way to store info, I think we need move it to connection_info.